### ISSUE TITLE
Study the WebID profile as the entry point for data discovery

### ISSUE THREAD

**Issue by lecoqlibre** [Wimmics/solid-start#16]
I started to think how the WebID profile(s) could be used to discover data, see https://github.com/datafoodconsortium/standard/discussions/4#discussioncomment-7635937.

Note: WebID profile can be used for any social agent being an individual or an organization.

Users can even have multiple profiles, here is some thoughts about it:
Regarding the DFC case, this could be used to allow a same user to use different workflows in order to manage data in different contexts like for instance:
- his own farm;
- the shop of a group of producers he is member of;
- and a public online shop.

Here is an example of an extended profile linked to a preferences file:
```turtle
<> a foaf:PersonalProfileDocument;
    foaf:primaryTopic <https://webid.provider/user/card#me>.

<https://webid.provider/user/card#me> a dfc-b:Person;
    pim:preferencesFile <link/to/the/extended/preferences>.
```

**Comment by FabienGandon** [comment]
if your prefix pim: corresponds to the Workspace Ontology http://www.w3.org/ns/pim/space#

I see that
```turtle
   ws:ConfigurationFile     :subClassOf gen:InformationResource .
```
and
```turtle
    ws:preferencesFile     a rdf:Property;
         :domain foaf:Agent;
         :label "preferences file";
         :range ws:ConfigurationFile .
```

but I am not sure what is meant by "preferences" and why we are talking about preference 
```turtle
<https://webid.provider/user/card#me> a dfc-b:Person;
    pim:preferencesFile <link/to/the/extended/preferences>.
```

as opposed to using for instance :
```turtle
    ws:storage     a rdf:Property;
         :comment """The storage in which this workspace is, or the storage which
contains this resource, or a storage available to this agent to use.""";
         :label "storage";
         :range ws:Storage .
```

is the use of `pim:preferencesFile` and preferences file to provide pointers to any type of additional data not only preferences?

**Comment by lecoqlibre** [comment]
> is the use of pim:preferencesFile and preferences file to provide pointers to any type of additional data not only preferences?

Yes, I'm following the [Solid WebID profile spec](https://solid.github.io/webid-profile/#private-preferences) which says:
> The preference Document can include settings and preferences, and pointers to the owner's personal, private data, be it contacts, pictures or health data.

**Comment by FabienGandon** [comment]
so it means it is the content of the `ws:ConfigurationFile` that will contain pointers to other sources but not directly the value of the property `pim:preferencesFile` right?

**Comment by lecoqlibre** [comment]
> so it means it is the content of the ws:ConfigurationFile that will contain pointers to other sources but not directly the value of the property pim:preferencesFile right?

Yes.
